spec/02-integration/03-db/01-db_spec.lua

```diff
@@ -662,6 +662,57 @@ for _, strategy in helpers.each_strategy() do
       assert.equal(any, opts.serial_consistency)
     end)
 
+    cassandra_only("does support different write and read consistency configration options", function()
+      local quorum = require "cassandra".consistencies.quorum
+      local three = require "cassandra".consistencies.three
+
+      local conf = utils.deep_copy(helpers.test_conf)
+
+      conf.cassandra_contact_points = { "localhost" }
+      conf.cassandra_consistency_proxy = {
+        "read:QUORUM",
+        "write:THREE",
+        read  = "quorum",
+        write = "three",
```

Comment:
I understand why `read = "quorum", write = "three"` are there - that's how they look after the config has been parsed. Why are `"read:QUORUM"` and `"write:THREE"` here too?